synapse/rest/media/v1/oembed.py

```diff
@@ -154,11 +154,14 @@ def parse_oembed_response(self, url: str, raw_body: bytes) -> OEmbedResult:
                 "og:url": url,
             }
 
-            # Use either title or author's name as the title.
-            title = oembed.get("title") or oembed.get("author_name")
+            title = oembed.get("title")
             if title:
                 open_graph_response["og:title"] = title
 
+            author_name = oembed.get("author_name")
+            if author_name:
+                open_graph_response["og:author_name"] = author_name
```

Comment:
`og:author_name` doesn't seem to be a thing as far as I can tell?

Reply:
Right, author_name is from oEmbed... I can't find a property where it would fit, and I couldn't find in the standard something that says that it's okay to add your own properties.

Here are the solutions I can think of:

- Leave it like that, with that non-standard property
- Delete `og:author_name` from the dict in the caller function in `preview_url_resource`. Not a pretty solution IMHO.
- Drop the feature that takes the title as author when title is null
- Refactor this code so that the oEmbed preview returns a custom struct instead of an OpenGraph dict and have `preview_url_resource` translate that to OpenGraph. I'm not sure what would be the impact of that, maybe that's a lot of work.

Please tell me what you think, or if you have other ideas.

Reply:
Hi again, I implemented something else in the end. I moved out the author_name out of the OpenGraph response and to the OEmbedResult class. Now we fully respect the OpenGraph standard.

I pushed my changes in the last commit, nevermind the forced push, I only fixed that last commit and didn't change what you have already reviewed.

Please tell me what you think of this :)

Reply:
Ah, thanks! Sorry for not responding, read the notification on the wrong device and...failed. 👍 I think that change makes sense.